packages/n4s/src/rules/isValueOf.ts

```diff
@@ -0,0 +1,13 @@
+import bindNot from 'bindNot';
+import { isNotNullish } from 'isNullish';
+import { isString, isPlainObject } from 'lodash';
+
+export function isValueOf(value: string, objectToCheck: object): boolean {
+  return (
+    isNotNullish(objectToCheck) &&
+    isPlainObject(objectToCheck) &&
```

Comment:
There's no real reason we should only allow object literals. For all I care, we can also detect properties of all sort of objects.
```suggestion
```

Reply:
removed plain object check